src/app/add-new-course/add-new-course.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-add-new-course',
+  templateUrl: './add-new-course.component.html',
+  styleUrls: ['./add-new-course.component.scss']
+})
+export class AddNewCourseComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```

Comment:
It this case it makes no sense to implement `OnInit` interface. So please, delete `ngOnInit` method to make the component less verbose. The constructor can be deleted as well.